packages/material-ui-lab/src/ToggleButton/ToggleButton.js

```diff
@@ -30,6 +30,7 @@ export const styles = theme => ({
       '& + &': {
         borderLeft: 0,
         marginLeft: 0,
+        minWidth: 48
```

Comment:
We need a different approach, it breaks with overrides

Reply:
Is `minWidth` different from `borderLeft`/`marginLeft` because we expect users to override the former but not the later?

Reply:
What about the small and large size? What about people changing the default width? I have been too strong with "we need a different approach". Still, I think that we should explore the alternatives we have.